src/account-info/AccountDetails.js

```diff
@@ -46,8 +48,12 @@ class AccountDetails extends PureComponent<Props> {
   static contextType = ThemeContext;
   context: ThemeColors;
 
+  handlePress = () => {
+    this.props.dispatch(navigateToAccountDetails(this.props.user.bot_owner_id || 0));
```

Comment:
@ray-kraesig Typescript has a great `as` keyword that allows you to force a type, so for example if `x` is `Bot | void` It can be used as `const y = x as Bot`.

Does Flow have a similar keyword?